library/Solarium/QueryType/Select/ResponseParser/Component/Grouping.php

```diff
@@ -67,7 +67,9 @@ public function parse($query, $grouping, $data)
             // parse field groups
             $valueResultClass = $grouping->getOption('resultvaluegroupclass');
             $documentClass = $query->getOption('documentclass');
-            foreach ($grouping->getFields() as $field) {
+
+            // check grouping fields as well as the grouping function (either can be used in the query)
+            foreach (array_merge($grouping->getFields(), [$grouping->getFunction()]) as $field) {
```

Comment:
I just see the current CI config allows 5.3 failures, that's why the tests pass. But the composer file still requires 5.3.2 and I cannot change that in a minor release.